README.md

```diff
@@ -193,6 +193,16 @@ More resources can be found:
 Examples for deployment via systemd and other use cases can be found in the [deployment
 repository](https://github.com/coredns/deployment).
 
+## Deprecation Policy
+
+When there is a backwards incompatible change in CoreDNS the following process is followed:
+
+* Release x.y.z: Announce that in the next release we will make backward incompatible changes.
+* Release x.y+1.0: Make the changes, but allow the old configuration to be parsed. I.e. CoreDNS will
+  start from an unchanged Corefile.
+* Release x.y+1.1: Remove the lenient parsing, so CoreDNS will not start if those features are still
```

Comment:
is this another +1, for a total of +2.1 from the original x.y.z ?  ...

Or are we going from ignored, to breakage in a minor release increment?

Reply:
NM, i was confused by the notation.